src/app/home/home.component.html

```diff
@@ -1,55 +1,35 @@
-
-<!-- Image and text -->
-<nav class="navbar navbar-primary bg-info">
-  <a class="navbar-brand" href="#">
-    <img src="../../assets/logo.png" width="30" height="30" class="d-inline-block align-top" alt="">
-    Turntabl
-  </a>
-</nav>
-
-<div class="text-center" *ngIf="msgShow == true" > 
+<!-- Success message -->
+<div class="text-center mes" *ngIf="msgShow == true" > 
   <div class="alert alert-danger" *ngIf="valid == false" role="alert">
-    {{ msg }}
+    {{ message }}
   </div>
   <div class="alert alert-success" *ngIf="valid == true" role="alert">
-    {{ msg }}
+    {{ message }}
   </div>
-</div> 
+</div>
 
+<!-- Main form -->
+<div class="cardb col">
 <div class="fm form-container d-flex align-items-center col-6">
-  <form [formGroup]="form" (ngSubmit)="submit()">
-
-    <div class="align-items-center row">
-        <button class="btn btn-lg btn-success btn-center" >submit</button>
-    </div>
-
-    <div *ngFor="let choice of rolesList; let i=index" class="form-group row car">
-
-         <div class="card ca btn btn-default"  style="width: 22rem;">
-          <div class="card-body">
-            <h6 class="card-title">{{choice.roleName}}</h6>
-            <h6 class="card-subtitle mb-2 text-muted">Description</h6>
-            <p class="card-text">{{choice.description}}</p>
-            <input type="checkbox" [value]="choice.roleARN"  class="form-check-input form-control chk" (change)="onCheckChange($event)" id="exampleCheck1">
-          </div>
-        </div>
-
-    </div>   
-
-    <div class="align-items-center row">
-      <button class="btn btn-lg btn-success btn-center" >submit</button>
-    </div>
-
+  <form [formGroup]="form"(ngSubmit)="submit()">
+    
+     <button class="btn btn-lg btn-primary btn-center" type="submit">Submit</button>
+     
+    <div *ngFor="let choice of rolesList; let i=index" class="form-group row car checkbox">
```

Comment:
let's not embed the word "List" in  rolesList, let's just call it roles.
also let's rename "choice" to role'.

for person in people
for building in buildings
for role in roles